src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    class="form">
+      <fieldset class="block">
+        <legend class="title">
+          Personal information
+        </legend>
+        <div class="element">
+          Surname:
+          <input required
+           type="text"
+           autocomplete="off"
+          name="Surname"
+          >
+        </div>
+        <div class="element">
+          Name:
+          <input required
+           type="text"
+           autocomplete="off"
+           name="Name"
+          >
+        </div>
+        <div class="element">
+        How old are You?
+          <input required type="Number"
+         name="Age"
+         max="100"
+         min="1"
+         value="12"
+          >
+        </div>
+        <div class="element">
+        Full date of birth:
+          <input required
+          type="date" name="dateOfBirth"
+          >
+        </div>
+        <label class="element">
+        I accept the term of the agreement
+          <input required type="checkbox" >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend class="title">
+          Registration
+        </legend>
+        <div class="element">
+          E-mail:
+          <input required
+          type="email"
+          name="Email"
+          placeholder="email@example.com"
+          >
+        </div>
+        <div class="element">
+          Password:
+          <input required
+          type="password"
+          name="password"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend class="title">
+          An interesting fact about you!
+        </legend>
+        <div  class="element">
+          Do you love cats?
+          <label>
+            <input required
+            type="radio"
+            name="LoveCats"
+            value="yes"
+            >
+            Yes
+          </label>
+          <label class="element">
+            <input required type="radio"
+            name="LoveCats"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+        <div class="element">
+        What is your favorite color?
+          <input
+          required
+          type="color"
+          name="color"
+          >
```

Comment:
```suggestion
          <input
            required
            type="color"
            name="color"
          >
```

Reply:
check your code style according to the checklist
<img width="914" alt="image" src="https://user-images.githubusercontent.com/102411130/224252139-46e8d486-ef75-4ac7-b6d9-4efb292dd33c.png">

Reply:
fix all such cases

Reply:
> 

This comment has not been fixed. Please, check your code style against checklist